cmd/collaborators/add.go

```diff
@@ -181,40 +180,7 @@ func promptCollaboratorsAddSlackUserPrompts(
 	return slackUser, nil
 }
 
-// promptCollaboratorsAddPermissionPrompts gathers the collaborator permission
-// from selection if the experiment allows
-func promptCollaboratorsAddPermissionPrompts(
```

Comment:
Changed the behaviour here: before when adding a new collaborator we required the permission type to be set, but I think we can default to 'owner' unless otherwise specified

Reply:
Thanks!